extend-admin-console-quick-start/README.md

```diff
@@ -0,0 +1,44 @@
+# Customizing Keycloak Themes
+
+The Keycloak Admin Console, which is the interface for managing Keycloak, is built with [PatternFly](https://www.patternfly.org/v4/) - a set of UI design patterns and resources that can be used to create consistent, user-friendly interfaces for web applications.
+
+## Creating Themes
+
+With themes, it is possible to configure the Admin Console's look and feel. To ensure consistency and maintainability of the custom Keycloak theme, we advise using PatternFly best practices when customizing the Keycloak Admin Console. This includes following the recommended folder structure and using PatternFly CSS classes and guidelines when creating custom styles and overrides.
+
+### Required Folder Structure
+
+The recommended folder structure for a custom Keycloak theme is as follows:
+
+    ├── theme
```

Comment:
`theme` can be any string.  It probably wouldn't be called `theme`.  Suggest you change this to something like `my-cool-theme-name`.

```suggestion
    ├── my-cool-theme-name
```

Reply:
Hi @ssilvert I have addressed your last comment. Can we merge this now?